tgbot/src/types/message/mod.rs

```diff
@@ -276,18 +277,34 @@ pub enum EditMessageResult {
     Bool(bool),
 }
 
-#[derive(Debug, failure::Fail, derive_more::From)]
+#[derive(Debug, derive_more::From)]
```

Comment:
I didn't saw `derire_more::From` for errors above

Reply:
Sometimes we need to convert error explicitly